speech/google/cloud/gapic/speech/v1/speech_client_config.json

```diff
@@ -6,9 +6,7 @@
           "DEADLINE_EXCEEDED",
           "UNAVAILABLE"
         ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+        "non_idempotent": []
```

Comment:
I have not been able to chase this down, and this needs to go out today for a Monday announcement. I can make the change later if it ends up being wrong.